rllib/tuned_examples/ppo/pendulum-ppo-with-rl-module.yaml

```diff
@@ -19,5 +19,7 @@ pendulum-ppo:
         observation_filter: MeanStdFilter
         seed: 42
         enable_connectors: true
+        model:
+            fcnet_activation: relu
```

Comment:
When you first introduced this test, the tf models used relus regardless of configuration.
Together with this fixed seed this made for an apparently advantagious situation.

The standard for activation function is actually tanh all over RLlib afaics (I wonder why that is, sounds pretty slow).
In order to make this test pass, I tried to reproduce the exact model that we used when you introduced the test, hence this config value.

Reply:
I looked at this, a little bit. I think the tf thing is slow not because of activation but because of sth else. I think we should disable this test for now and re-enable when eager-tracing works on the rllib sampling side. So for now I would just comment the tf part of the test out and move on.